jormungandr/src/topology/quarantine.rs

```diff
@@ -62,24 +68,30 @@ impl ReportRecords {
     }
 
     /// Returns whether the node has been quarantined or not.
-    pub fn report_node(&mut self, topology: &mut poldercast::Topology, node: Peer) -> bool {
+    pub fn report_node(
+        &mut self,
+        topology: &mut poldercast::Topology,
+        node: Peer,
+    ) -> ReportNodeResult {
```

Comment:
Bikeshed: don't like "result" naming, should not commingle with `Result`. Maybe `ReportNodeStatus`?